client/deployments/client.go

```diff
@@ -199,8 +204,8 @@ func (c *Client) UploadArtifact(description, artifactPath, tokenPath string, noP
 		defer pW.Close()
 		defer artifact.Close()
 
-		writer.WriteField("size", strconv.FormatInt(artifactStats.Size(), 10))
-		writer.WriteField("description", description)
+		_ = writer.WriteField("size", strconv.FormatInt(artifactStats.Size(), 10))
+		_ = writer.WriteField("description", description)
```

Comment:
should we add error handling here?

Reply:
@kjaskiewiczz Actually, I first added error handling in these, but if you look next line, `CreateFormFile` error is already explicitly ignored so I concluded to ignore these previous ones too.